### DISCUSSION TITLE
Change youtube video quality while the video is playing?

### DISCUSSION THREAD

**Discussion by rp1231** [smplayer-dev/smplayer#491] [Q&A]
I want to change the video quality of a youtube video I have loaded while the video is playing.
Something like right click and choose video playback quality.
Like how you can choose the video quality on youtube while a video is playing.
Is that possible?

**Comment by smplayer-dev** [comment]
No, it's not possible.

**Reply by tomasz1986** [reply]
It would be great to have this feature though 🙂. Some other (proprietary) players do offer this. You can basically switch on the fly between every option that is listed when running `youtube-dl -F` on a video. It's extremely handy, e.g. when the connection is too slow to play the higher quality video or when you want to listen to the audio only, etc.
